extensions/spark/kyuubi-extension-spark-common/src/main/antlr4/org/apache/kyuubi/sql/KyuubiSparkSQL.g4

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-grammar ZorderSqlExtensions;
+grammar KyuubiSparkSQL;
```

Comment:
I rename it to `KyuubiSparkSQL` because 1) we use the upper case `SQL` anywhere other than this place 2) we may introduce `KyuubiSQL` in server module in the future.